docs/zarr-v3.md

```diff
@@ -0,0 +1,74 @@
+
+```
+uv pip install zarrs
+```
+
+```python
+import zarr
+import zarrs
+zarr.config.set({"codec_pipeline.path": "zarrs.ZarrsCodecPipeline"})
+```
+
+However, this pipeline is not compatible with all types of zarr store, especially remote stores and there are limitations on where rust can give a performance boost for indexing.  We therefore recommend this pipeline for writing full datasets and reading contiguous regions of said written data.
+
+## Codecs
+
+The default `zarr-python` v3 codec for the [v3 format] is no longer `blosc` but `zstd`.  While `zstd` is more widespread, you may find its performance to not meet your old expectations.  Therefore, we recommend passing in the [`BloscCodec`] to `compressor` on {func}`~anndata.AnnData.write_zarr` if you wish to return to the old behavior.
+
+There is currently a bug with `numcodecs` that prevents data written from other non-numcodecs `zstd` implementations from being read in by the default zarr pipeline (to which the above rust pipeline falls back if it cannot handle a datatype or indexing scheme, like `vlen-string`): https://github.com/zarr-developers/numcodecs/issues/424.  Thus is may be advisable to use `BloscCodec` with `zarr` v3 file format data if you wish to use the rust-accelerated pipeline until this issue is resolved.
+
+The same issue with `zstd` applies to data that may eventually be written by the GPU `zstd` implementation (see below).
+
+## GPU i/o
+
+At the moment, it is unlikely your `anndata` i/o will work if you use [`zarr.enable_gpu`].  It's *possible* dense data i/o i.e., using {func}`anndata.io.read_elem` will work as expected, but this functionality is untested - sparse data, awkward arrays, and dataframes will not.  `kvikio` currently provides a [`GDS`-enabled store] although there are no working compressors at the moment exported from the `zarr-python` package (work is underway for `Zstd`: https://github.com/zarr-developers/zarr-python/pull/2863).
+
+We anticipate enabling officially supporting this functionality officially for dense data, sparse data, and possibly awkward arrays in the next minor release, 0.13.
+
+## Asynchronous i/o
+
+At the moment, `anndata` exports no `async` functions.  However, `zarr-python` has a fully `async` API and provides its own event-loop so that users like `anndata` can interact with a synchronous API while still beenfitting from `zarr-python`'s asynchronous functionality under that API.  We anticipate providing `async` versions of {func}`anndata.io.read_elem` and {func}`anndata.experimental.read_dispatched` so that users can download data asynchronously without using the `zarr-python` event loop.  We also would like to create an asynchronous partial reader to enable iterative streaming of a dataset.
+
+
+[v3 format]: https://anndata.readthedocs.io/en/stable/generated/anndata.settings.html#anndata.settings.zarr_write_format
```

Comment:
No, never link to anything with a sphinx inventory using regular links. This includes everything on readthedocs, and most other Python projects that have a docs page.

We want to automatically have links update when things move around and to be notified when links break, which is only possible when you use intersphinx for linking.

You can use this to find links: https://sphobjinv.readthedocs.io/en/latest/cli/suggest.html

Reply:
What should we do here because this attribute doesn't exist yet? So my solution was to use the link here and then we could change it once the feature is on `stable`

Reply:
Maybe this should be `latest` not `stable` but the point still stands (there's no inventory link)

Reply:
I think I need to explain this to you in person, I can’t really respond to this without explaining how readthedocs works on a fundamental level.